nikola/data/themes/base-jinja/templates/comments_helper_disqus.tmpl

```diff
@@ -30,7 +30,7 @@
 
 {% macro comment_link(link, identifier) %}
     {% if comment_system_id %}
-    <a href="{{ link }}#disqus_thread" data-disqus-identifier="{{ identifier }}">Comments</a>
+    <a href="{{ link }}#disqus_thread" data-disqus-identifier="{{ identifier }}">${messages(Comments)}</a>
```

Comment:
You're missing quotes around the string. Moreover, Jinja2 has different syntax to Mako.